packages/core/src/components/OnboardingNavigationMainItem.svelte

```diff
@@ -44,9 +51,13 @@
   </span>
 </div>
 
+<div class="visahoi-edit-mode-button">
+  <button on:click={toggleEditMode}>{buttonLabel}</button>
```

Comment:
Please make the button a bit nicer, it doesn't fit to our visahoi design currently